src/components/Profile.js

```diff
@@ -0,0 +1,45 @@
+import { useSelector } from 'react-redux';
+import React from 'react';
+import MyProfile from './MyProfile';
+
+function Profile() {
+  const rockInfo = useSelector((state) => state.rockets);
+  let count = 0;
+  return (
+    <div className="row">
+      <div className="col-5 offset-1">
+        <MyProfile />
+      </div>
+      <div className="col-5">
+        <h1>My Rockets</h1>
+        <div>
+          {/* eslint-disable-next-line */}
```

Comment:
Kindly remove this disabled linter, linting improves readability, removes errors. 👍

Please check for other occurrences 🙂

Reply:
Thanks for the suggestion. All of them have been removed.